src/js/sticky-nav.js

```diff
@@ -1,12 +1,13 @@
-export function initStickyNav() {
+function stickToTop() {
   const nav = document.querySelector('.main-header');
+  const scroll = window.scrollY;
```

Comment:
Since you’re using this `const` only one in the code, there is no need to have this assignment. You can just simply use `window.scrollY` in the condition for `isScrolled`